papers/tests/test_api.py

```diff
@@ -126,3 +126,81 @@ def test_bibtex_formatting(self):
             self.assertEqual(resp.status_code, 200)
             self.assertEqual(resp.content.decode('utf-8').strip(),
                              bibtex.strip())
+
+    def test_bibtex_formatting_researcher(self):
+        bibtex_output = """@inproceedings{Amarilli2015,
+  author = {Amarilli, Antoine},
+  doi = {10.1145/2744680.2744690},
+  journal = {Proceedings of the 2015 ACM SIGMOD on PhD Symposium - SIGMOD '15 PhD Symposium},
+  month = {jan},
+  title = {Structurally Tractable Uncertain Data},
+  url = {https://doi.org/10.1145/2744680.2744690},
+  year = {2015}
+}
+
+@inproceedings{Amarilli2015_2,
+  author = {Amarilli, Antoine and Benedikt, Michael},
+  doi = {10.1109/lics.2015.37},
+  journal = {2015 30th Annual ACM/IEEE Symposium on Logic in Computer Science},
+  month = {jul},
+  title = {Finite Open-World Query Answering with Number Restrictions},
+  url = {https://doi.org/10.1109/lics.2015.37},
+  year = {2015}
+}"""
+        from django.core.management import call_command
+        call_command('clear_index', interactive=False)
+        r1 = Researcher.create_by_name('Antoine', 'Amarilli')
+        p1 = Paper.create_by_doi('10.1109/lics.2015.37')
+        p1.set_researcher(0, r1.id)
+        p2 = Paper.create_by_doi('10.1145/2744680.2744690')
+        p2.set_researcher(0, r1.id)
+        call_command('rebuild_index', interactive=False)
```

Comment:
At the moment, if you do not explicitly index papers in ElasticSearch with `p1.update_index()`, it is normal that they do not end up there. So I would expect that you should be able to replace `call_command` by `p1.update_index()` and `p2.update_index()`.